src/components/policy/policy_external/src/cache_manager.cc

```diff
@@ -1035,16 +1035,22 @@ bool CacheManager::SetCountersPassedForSuccessfulUpdate(
 int CacheManager::DaysBeforeExchange(uint16_t current) {
   LOG4CXX_AUTO_TRACE(logger_);
   CACHE_MANAGER_CHECK(0);
+
+  const rpc::Optional<rpc::Integer<uint16_t, 0, 65535> > days_after_epoch =
```

Comment:
@LitvinenkoIra, perhaps better to use reference on `days_after_epoch` rather then copy